### PR TITLE
Sum of squares

### PR DESCRIPTION
**Pull request by drkennetz**
Challenge setup for #11

### CODE REVIEW

**Comment by ericwburden** [comment]
May want to include instructions in `challenge.md` for running tests/submitting solutions.

File: challenges/2021-05-18-sum-of-squares/solution_test.go [hunk 1]
```diff
@@ -0,0 +1,24 @@
+package solution
+
+import (
+	"fmt"
+	"testing"
+)
+
+func TestSumOfSquares(t *testing.T) {
+	tables := []struct {
```
Comment (birenderjit) [inline challenges/2021-05-18-sum-of-squares/solution_test.go:9]: Do we want to add a description for the tests and add more tests?

Comment (drkennetz) [reply challenges/2021-05-18-sum-of-squares/solution_test.go:9]: I didn't add a description for the tests. If this is a dealbreaker I might be able to get it pushed later today.

File: challenges/2021-05-18-sum-of-squares/go.mod [hunk 2]
```diff
@@ -0,0 +1,3 @@
+module solution
```
Comment (xanderyzwich) [inline challenges/2021-05-18-sum-of-squares/go.mod:1]: I believe that we usually include the problem statement in comment at the top of implementation file.

**Review by ericwburden — APPROVED** [review]
Minor quibbles (I'm approving anyway):

- In `challenge.md`, I'd include the command for running the tests in bullet #5 under 'Tackling this Challenge'
- As someone who knows even less Go than I do Java (which is very little), the test setup is a little opaque to me and I'd appreciate a few comments that would be totally unnecessary for anyone who knows any Go (probably). I'm less concerned about our Driver here, but about anyone who comes after and tries to tackle this challenge out-of-meetup and wants to understand what's being tested.

File: challenges/2021-05-18-sum-of-squares/go.mod (hunk 2 above)
Comment (ericwburden) [reply challenges/2021-05-18-sum-of-squares/go.mod:1]: Ooh, yes, this!

**Review by ericwburden — APPROVED** [review]

**Review by parhaml — APPROVED** [review]

**Pull request merged**